### ISSUE TITLE
MD031 not triggered in indented lists

### ISSUE THREAD

**Issue by ranebrown** [DavidAnson/markdownlint#189]
When using fenced code blocks inside a list the 'MD031 - Fenced code blocks should be surrounded by blank lines' rule is not triggered due to the leading indentation.

```markdown
- item 1
- item 2
    ```c
    int main()
    {
    return 0;
    }
    ```
- item 3
```

**Comment by DavidAnson** [comment]
It looks to me like MD031 *is* triggered for the top and bottom fences of your example: https://dlaa.me/markdownlint/#%25m%23%20Issue%20189%0A%0A-%20item%201%0A-%20item%202%0A%20%20%20%20%60%60%60c%0A%20%20%20%20int%20main()%0A%20%20%20%20%7B%0A%20%20%20%20return%200%3B%0A%20%20%20%20%7D%0A%20%20%20%20%60%60%60%0A-%20item%203%0A

Could you please provide more detail about how to reproduce the issue?

**Comment by ranebrown** [comment]
I'm using [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli). I was using version  0.13.0 and seeing this issue but updating to the latest (0.16.0) seems to have resolved the problem. Sorry for the trouble.

**Comment by DavidAnson** [comment]
No worries, glad you sorted it out!
